diff --color -ruN original/custom/dependencies/libzlib/gzlib.c modified/custom/dependencies/libzlib/gzlib.c
--- original/custom/dependencies/libzlib/gzlib.c	2025-01-10 19:42:43.000000000 -0700
+++ modified/custom/dependencies/libzlib/gzlib.c	2025-01-14 18:55:31.877912121 -0700
@@ -4,7 +4,7 @@
  */
 
 #include "gzguts.h"
-
+#include <unistd.h>
 #if defined(_WIN32) && !defined(__BORLANDC__)
 #  define LSEEK _lseeki64
 #else
diff --color -ruN original/custom/dependencies/libzlib/gzread.c modified/custom/dependencies/libzlib/gzread.c
--- original/custom/dependencies/libzlib/gzread.c	2025-01-10 19:42:43.000000000 -0700
+++ modified/custom/dependencies/libzlib/gzread.c	2025-01-14 18:55:35.998070054 -0700
@@ -4,7 +4,7 @@
  */
 
 #include "gzguts.h"
-
+#include <unistd.h>
 /* Local functions */
 local int gz_load OF((gz_statep, unsigned char *, unsigned, unsigned *));
 local int gz_avail OF((gz_statep));
diff --color -ruN original/custom/dependencies/libzlib/gzwrite.c modified/custom/dependencies/libzlib/gzwrite.c
--- original/custom/dependencies/libzlib/gzwrite.c	2025-01-10 19:42:43.000000000 -0700
+++ modified/custom/dependencies/libzlib/gzwrite.c	2025-01-14 18:55:41.173268296 -0700
@@ -4,7 +4,7 @@
  */
 
 #include "gzguts.h"
-
+#include <unistd.h>
 /* Local functions */
 local int gz_init OF((gz_statep));
 local int gz_comp OF((gz_statep, int));
